frameworks/autoxgboost/setup.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+HERE=$(dirname "$0")
+. $HERE/../shared/setup.sh
+if [[ -x "$(command -v apt-get)" ]]; then
+    SUDO apt-get update
+    SUDO apt-get install -y software-properties-common apt-transport-https libxml2-dev
+    SUDO apt-key adv --keyserver keyserver.ubuntu.com --recv-keys E084DAB9
```

Comment:
looks like this is importing the wrong public key:
```
root@a5753493cd7e:/bench# apt-key adv --keyserver keyserver.ubuntu.com --recv-keys E084DAB9
Executing: /tmp/apt-key-gpghome.uiTd5mx21z/gpg.1.sh --keyserver keyserver.ubuntu.com --recv-keys E084DAB9
gpg: key 4359ED62E084DAB9: public key "Totally Legit Signing Key <mallory@example.org>" imported
gpg: Total number processed: 1
gpg:               imported: 1
```
this causes later:
```
W: GPG error: https://cran.rstudio.com/bin/linux/ubuntu bionic-cran35/ InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 51716619E084DAB9
E: The repository 'https://cran.rstudio.com/bin/linux/ubuntu bionic-cran35/ InRelease' is not signed.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.
```

Reply:
probably a conflict between 2 keys with same suffix.
importing the key explicitly works better apparently:
```
root@a5753493cd7e:/bench# apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 51716619E084DAB9
Executing: /tmp/apt-key-gpghome.L0VQc8DfC6/gpg.1.sh --keyserver keyserver.ubuntu.com --recv-keys 51716619E084DAB9
gpg: key 51716619E084DAB9: public key "Michael Rutter <marutter@gmail.com>" imported
gpg: Total number processed: 1
gpg:               imported: 1
```

Reply:
@Coorsaa please change the key in your PR.
We must have the same issue with one or two other framework integrations, I'll take care of those.

Reply:
after this I have issues installing `r-cran-rjava`: see https://stackoverflow.com/questions/51256462/r-cannot-install-rjava-what-is-r-api-3-4

do you mind taking a look?